src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -62,8 +62,7 @@ public override Conversion GetMethodGroupDelegateConversion(BoundMethodGroup sou
                 // If synthesizing a delegate with `params` array, check that `ParamArrayAttribute` is available.
                 if (invoke.IsParams())
                 {
-                    Binder.AddUseSiteDiagnosticForSynthesizedAttribute(
-                        Compilation,
+                    checkWellKnownMemberAvailable(
```

Comment:
>checkWellKnownMemberAvailable [](http://example.com/codeflow?start=20&length=29)

Why not use the `Binder.AddUseSiteDiagnosticForSynthesizedAttribute` method?

Reply:
Updated, thanks. (I'm not sure why I'd overlooked `AddUseSiteDiagnosticForSynthesizedAttribute` previously.)